docs/faq.rst

```diff
@@ -1,20 +1,96 @@
 Frequently-Asked Questions
 ==========================
 
-My script works when launched as ``python3 script.py`` but exits immediately or throws an error when launched from Brickman or as ``./script.py``
--------------------------------------------------------------------------------------------------------------------------------------------------
+Q: Why does my Python program exit quickly or immediately throw an error?
+    A: This may occur if your file includes Windows-style line endings
+    (CRLF--carriage-return line-feed), which are often inserted by editors on
+    Windows. To resolve this issue, open an SSH session and run the following
+    command, replacing ``<file>`` with the name of the Python file you're
+    using:
 
-This may occur if your file includes Windows-style line endings, which are often
-inserted by editors on Windows. To resolve this issue, open an SSH session and
-run the following command, replacing ``<file>`` with the name of the Python file
-you're using:
+    .. code:: shell
 
-.. code:: shell
+        sed -i 's/\r//g' <file>
 
-    sed -i 's/\r//g' <file>
+    This will fix it for the copy of the file on the brick, but if you plan to edit
+    it again from Windows, you should configure your editor to use Unix-style
+    line endings (LF--line-feed). For PyCharm, you can find a guide on doing this
+    `here <https://www.jetbrains.com/help/pycharm/2016.2/configuring-line-separators.html>`_.
+    Most other editors have similar options; there may be an option for it in the
+    status bar at the bottom of the window or in the menu bar at the top.
 
-This will fix it for the copy of the file on the brick, but if you plan to edit
-it again from Windows you should configure your editor to use Unix-style endings.
-For PyCharm, you can find a guide on doing this `here <https://www.jetbrains.com/help/pycharm/2016.2/configuring-line-separators.html>`_.
-Most other editors have similar options; there may be an option for it in the
-status bar at the bottom of the window or in the menu bar at the top.
+Q: Where can I find demo code?
```

Comment:
I like shortening the README and streamlining things, but we need to make sure people can still find this info, especially questions like this one. Honestly I think demo code might be best in the main README, although others might be OK.

Reply:
Agreed. Moved the Demo Code section back to README.